ogre_mesh_viewer.py

```diff
@@ -643,6 +662,11 @@ def setup(self):
         camnode = scn_mgr.getRootSceneNode().createChildSceneNode()
         camnode.attachObject(self.cam)
 
+        self.grid_floor = grid.GridFloor(scn_mgr, "BaseWhiteAlphaBlended")
+        self.grid_floor.create_planes()
+        if self.grid_visible is True:
+            self.grid_floor.show_plane(self.fixed_yaw_axis)
```

Comment:
changing fixed_yaw_axis via UI is not handled

Reply:
It is...

Reply:
ah.. found it. If we do it in set_orientation below, we dont have to do it here though

Reply:
We have to, because of reloading....

When you load a new file with the open dialog the `MeshViewer::__init__()` function does not get called, but `MeshViewerGui::__init__()` does.

So there are options that get reset and others that don't.

If I don't call `set_orientation()` in `setup()` then when loading a new file the current orientation won't be respected.